docs/src/user_docs.md

```diff
@@ -1,3 +1,5 @@
 # User Documentation
 
-TODO
+
+- [Miden Assembly](./miden_assembly.md)
+- [Miden Standard Library](./miden_stdlib.md)
```

Comment:
I think these links will need to be updated once we update the structure as described in other comments. At the very least, the Miden Assembly link seems to be broken now.

Reply:
yes fixed